cypress/integration/modules/core.js

```diff
@@ -0,0 +1,97 @@
+/// <reference types="cypress" />
+
+context('Core', () => {
+  describe('centeredSlides & slidesPerView', () => {
+    beforeEach(() => {
+      cy.swiperPage();
+      cy.initSwiper({
+        centeredSlides: true,
+        slidesPerView: 3,
+        spaceBetween: 10,
+        navigation: {
```

Comment:
Not sure why it happens, but specifying navigation here will throw an error. Because somehow it looses default parameters (e.g. rest navigation params like `swiper.params.navigation.disabledClass` will be undefined)